docs/api/scheduler.md

```diff
@@ -591,7 +591,7 @@ In the example below there are three allocations belonging to two applications,
 
 ### Partition/Queue application
 
-Fetch an Application given a Partition, Queue(optional) and Application ID and displays general information about the application like used resources, queue name, submission time and allocations.
+Fetch an Application given a Partition, Queue(optional) and Application ID and displays general information about the application like used resources, queue name, submission time and allocations. In case queue name contains any special characters, needs to be url escaped to avoid issues.
```

Comment:
Nit: "In case the queue name contains any special characters, it needs to be url escaped to avoid issues."